### PR TITLE
config pull, ls, new DynamoDB schema

### PR DESCRIPTION
**Pull request by prog893**
- Implement config ls
- Implement config pull
- New DynamoDB Schema

### CODE REVIEW

File: aws_eden_cli/dynamodb.py [hunk 1]
```diff
@@ -167,11 +178,62 @@ def fetch_all_environments(table):
...
+        else:
+            logger.error(f"Unknown exception raised: {e}")
+            return None
+
+    if 'Items' not in r or len(r['Items']) == 0:
+        logger.warning(f"Profile {profile_name} not found in remote table!")
+        return None
+    elif len(r['Items']) != 1:
+        logger.warning(f"Got multiple profiles for profile name {profile_name} from remote table!")
+        return None
```
Comment (toVersus) [inline aws_eden_cli/dynamodb.py:224]: How about returning the profile with first or last match in remote table?

Comment (prog893) [reply aws_eden_cli/dynamodb.py:224]: Order on multiple items is not guaranteed AFAIK, thus I think we should fail here.

Comment (prog893) [reply aws_eden_cli/dynamodb.py:224]: This should not be normally happening. I wanted to see when this happens in wild actually.

**Review by toVersus — APPROVED** [review]
LGTM

**Pull request merged**